app/src/main/java/com/yalta/activities/fragments/AdminBrowseFragment.kt

```diff
@@ -30,4 +34,26 @@ class AdminBrowseFragment : Fragment() {
 
         return binding.root
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        pointSearch.clearFocus()
```

Comment:
Id' rename `pointSerach` to `pointSearchField`/`Input` for clarity

Reply:
done